ui/app/components/ui/page-container/index.scss

```diff
@@ -91,24 +92,24 @@
   }
 
   &__back-button {
+    @include Paragraph;
+
     color: #2f9ae0;
-    font-size: 1rem;
     cursor: pointer;
-    font-weight: 400;
   }
 
   &__title {
+    @include H2;
+
     color: $black;
-    font-size: 2rem;
     font-weight: 500;
-    line-height: 2rem;
     margin-right: 1.5rem;
   }
 
   &__subtitle {
+    @include H6;
```

Comment:
this results in a small change to font size. @Gudahtt also reviewed this change here:

https://github.com/MetaMask/metamask-extension/pull/9074/files?file-filters%5B%5D=.scss#r466433436